test/close.test.ts

```diff
@@ -40,17 +37,37 @@ describe('StakingPool.closePool', () => {
   beforeEach(async () => {
     testEnv = await loadFixture(fixture);
     actions = createTestActions(testEnv);
-    await actions.faucetAndApproveReward(deployer, MAX_UINT_AMOUNT);
+    await actions.faucetAndApproveReward(deployer, constants.MaxUint256.toString());
```

Comment:
`MaxUint256` doesn't need to be converted to string beacuse the type of input is BigNumberish.

Reply:
or How about making action functions receive both string type and bignumber type?

Reply:
It seems better to change all helperfunctions to take BigNumberish `amount`.

Reply:
absolutely 👍